tool/icon_generator/analysis_options.yaml

```diff
@@ -1 +1,5 @@
-include: package:flutter/analysis_options_user.yaml
```

Comment:
Thanks for the updates! I haven't seen things using this older lint config file in a while; perhaps we should set this repo up to run its CI ~weekly (https://github.com/flutter/tools_metadata/blob/master/.github/workflows/build.yaml)?

Reply:
I've opened https://github.com/flutter/tools_metadata/pull/50